### PR TITLE
Merge MVP site

### PR DESCRIPTION
**Pull request by iamawatermelo**
- [x] Write policies
- [x] Email subscriptions
- [x] Email unsubscribe
- [x] Set production environment variables

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,2 +1,315 @@
...
+      <Gear />
+    </div>
+    <div class="gear third">
+      <Gear />
+    </div>
+  </div>
+
+  <div class="info-card">
+    <h2>What is a hackathon?</h2>
+    <p>A hackathon is where teenagers come together to build a project and share them with the world!</p>
```
Comment (iamawatermelo) [inline src/routes/+page.svelte:62]: Needs rewriting---I'm not good at copywriting...

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+    <DoubleArrow class="logo" />
+    <p class="header">Revolution</p>
+    <div class="content">
+      <ul>
+        <li><a href="https://hackclub.com/slack/">Hack Club Slack (talk to us!)</a></li>
+        <li><a href="https://github.com/Revolution-Hacks/revolution">Source code</a></li>
+        <li><a href="https://hcb.hackclub.com/revolution">Financials</a></li>
+      </ul>
+
+      <p>
```
Comment (iamawatermelo) [inline src/routes/+layout.svelte:28]: @devramsean0 Can you check that this is correct?

Comment (transcental) [reply src/routes/+layout.svelte:28]: That's correct

**Comment by devramsean0** [comment]
Slight overflow on mobile

**Comment by iamawatermelo** [comment]
TODO: [WCAG checklist](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_overview&levels=aaa&showtechniques=pageinfo%2C111#top):

- [x] Percievable
  - [x] 1.1 Text Alternatives
  - [x] 1.2 Time-based Media
  - [x] 1.3 Adaptable
  - [x] 1.4 Distinguishable
 - [x] Operable
   - [x] 2.1 Keyboard Accessible
   - [x] 2.2 Enough Time
   - [x] 2.3 Seizures and Physical Reactions
   - [x] 2.4 Navigable
   - [x] 2.5 Input Modalities
 - [x] Understandable
   - [x] 3.1 Readable
   - [x] 3.2 Predictable
   - [x] 3.3 Input Assistance
 - [x] Robust
   - [x] 4.1 Compatible

File: src/routes/+page.svelte (hunk 1 above)
Comment (transcental) [reply src/routes/+page.svelte:62]: this is fine for now, we can do a separate copy pass later

**Review by transcental — APPROVED** [review]
lgtm!

**Review by pixelglide — APPROVED** [review]

**Pull request merged**